data/import/common/common/glossary.csv

```diff
@@ -4227,3 +4227,5 @@ merchant_relationship.mail.trans.merchant_relationship_delete.main_text,"Merchan
 merchant_relationship.mail.trans.merchant_relationship_delete.main_text,"Der Händler %merchant_name% hat Ihre Händlerbeziehung beendet. Bei Fragen wenden Sie sich bitte an den Händler.",de_DE
 merchant_relationship.mail.trans.merchant_relationship_delete.link,Contact merchant,en_US
 merchant_relationship.mail.trans.merchant_relationship_delete.link,Händler kontaktieren,de_DE
+dynamic_entity.validation.invalid_url,The URL is invalid. `%errorPath%` field '%fieldName%' must have a URL data format.,en_US
```

Comment:
https://github.com/spryker/suite-nonsplit/pull/8845/files
additionally `src/Pyz/Zed/DynamicEntity/data/installer/configuration.json` has to be integrated